### DISCUSSION TITLE
Altiverb plugin fails in AudioGridder

### DISCUSSION THREAD

**Discussion by Sequence1** [apohl79/audiogridder#1282] [Help]
Hello. I have been using AudioGridder for about a year, and I am AMAZED at how well it works. Seamless, always reliable... and it saves me a tonne of time!

I just purchased the Mac Studio M2 Ultra... Because all my plugins are not Silicon native, I am forced to run Logic in Rosetta mode. I also run AudioGridder 1.2 Server in Rosetta. 

However, Altiverb 7 fails when scanning my plugins. BTW, Altiverb says it is NOT Silicon native. I assume it should work... but it doesn't

Can anyone help me find a solution?

Very best Wishes from Canada

Angelo

Mac Studio Ultra
192 GB Ram

**Comment by apohl79** [comment]
I assume altiverb worked before?

You can check the altiverb binary first, to see if it contains an x86_64 payload:

The following command is an example, change the Revolver path to the path of the altiverb plugin:

`lipo -archs /Library/Audio/Plug-Ins/VST3/Revolver.vst3/Contents/MacOS/Revolver`

you should see the following, if it is a universal binary:

`x86_64 arm64`

**Comment by Sequence1** [comment]
Thank you for the quick response Andreas!

Yes, it used to work beautifully on my old machine

I’m not the most computer literate user, so I hope I did what you asked me correctly… 

I followed the path, and once I got to the plugin, I went into contents, maac os and, I opened the plugin in text edit…There is no x86_64 arm64 in the code. I did this in both vst and au plugins.
Did I do that correctly?

You should also be aware that there is no vst3 plugin, just vst, au, MAS, AAX.

Let’s assume it’s not universal binary, what does that mean? What should I/ could I do next?

Many thanks for your software, and you attention to my issue.

Best

Angelo

**Comment by Sequence1** [comment]
I’m sorry to bother you again…

But I really need to get this all working. In case you didn’t get my response to your comment it is as follows:

Thank you for the quick response Andreas!

Yes, it used to work beautifully on my old machine

I’m not the most computer literate user, so I hope I did what you asked me correctly… 

I followed the path, and once I got to the plugin, I went into contents, maac os and, I opened the plugin in text edit…There is no x86_64 arm64 in the code. I did this in both vst and au plugins.
Did I do that correctly?

You should also be aware that there is no vst3 plugin, just vst, au, MAS, AAX.

Let’s assume it’s not universal binary, what does that mean? What should I/ could I do next?

Many thanks for your software, and you attention to my issue.

Again, it was working fine on my old machine (Mojave)

Best wishes

Angelo

**Reply by apohl79** [reply]
The command I mentioned had to be executed in the terminal app. It will show the available architectures in an executable. You can also run it on the VST. 

The point is, if you run AG server in rosetta, the plugin has to have an x86_64 binary and if you run AG natively, an arm64 binary is needed. Universal binaries (which contain both architectures) will work in both scenarios. 

With some manual work, you can run two instances of AG server, one for each architecture. You will have to search the discussions for this. (I'm just on the phone, as I'm traveling) In essence, you can give each server a different ID and run them in parallel. From the DAW side you will see both servers and can connect them individually.

I hope you get it solved.

**Comment by Sequence1** [comment]
Good day Andreas.

I deleted AG Universal, and then installed AG For Intel chip (10.8+).

This doesn’t recognize Altiverb either.

This version of AG was working very well on my old Intel machine.

I did wipe the cache and rescanned my plugins.

There must be something I’m doing wrong.

Not to be too dramatic, but this is essential to having my new system work.

Very best wishes

Angelo

**Reply by apohl79** [reply]
Did you try the arm64 version of AG as well? Or have you been able to verify that the failing plugin is x86_64 only?

Does AG attempt to scan the plugin? There is a scan log for each plugin, it does usually not show useful info, but you should check it anyways.

**Comment by Sequence1** [comment]
Hello Andreas

OK, so I went to Plugins list, selected the failed plugins (in red), chose “ALTIVERB.AU,/ .VST” and selected “remove from Blacklist”.. rescanned, 

and ALTIVERB is now showing up!

Many, many thanks!!

Best

A
